app.js

```diff
@@ -88,6 +88,23 @@ app.post("/watches/collection", async (req, res) => {
   });
 });
 
+// GET /Collection
+app.get("/watches/collection/id", async (req, res) => {
```

Comment:
Does this work to get the parameters? don't you need it to be `/watches/collection/:collectionID` instead?